includes/wsuwp-shortcode-fundselector.js

```diff
@@ -426,14 +426,16 @@ function continueAction(event)
 
 		if(designations.length === 1)
 		{
+			// Clear fields used by mulitple designations
+			jQuery("#iDonateEmbed").attr("data-custom_gift_amount", null);
```

Comment:
Use the [`removeAttr()` function in jQuery](https://api.jquery.com/removeAttr/) instead, so it actually removes it. I think that will prevent any issues from iDonate trying to process a null value.